### PR TITLE
Add ekinops for the first time

### PR DESCRIPTION
**Pull request by TOUFIKIzakarya**
- Add the ekinops parser for the first time

### CODE REVIEW

**Comment by vg-svitla** [comment]
Looks good to me! Can you please check if prettier is applied to jsons? It will automatically add end lines where necessary etc

**Review by vg-svitla — DISMISSED** [review]

File: Ekinops/ekinops-oneos/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## 2024-02-21
```
[the remark this answers is not in the document]
Comment (TOUFIKIzakarya) [reply Ekinops/ekinops-oneos/CHANGELOG.md:10]: yes you're right. I think we have to do that

**Comment by TOUFIKIzakarya** [comment]
Thanks @vg-svitla 👌🏻

**Review by vg-svitla — DISMISSED** [review]

File: Ekinops/ekinops-oneos/_meta/manifest.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+uuid: 4760d0bc-2194-44e5-a876-85102b18d832
+name: Ekinops OneOS
```
Comment (squioc) [inline Ekinops/ekinops-oneos/_meta/manifest.yml:2]:
```suggestion
name: Ekinops OneOS [BETA]
```

**Review by squioc — CHANGES_REQUESTED** [review]
LGTM.
Can you add the beta flag in the name of the format?

**Review by squioc — APPROVED** [review]

**Pull request merged**